dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -1203,9 +1219,19 @@ def test_report_remove_disabled_flag(self):
         # with options to disable FloatColumn `precision`
         # and with remove_disabled_flag == True
         profiler_options = ProfilerOptions()
-        profiler_options.set({"precision.is_enabled": False})
+        profiler_options.set(
+            {
+                "precision.is_enabled": False,
+                "structured_options.multiprocess.is_enabled": False,
+            }
+        )
         profiler = dp.StructuredProfiler(data=data, options=profiler_options)
-        report = profiler.report(report_options={"remove_disabled_flag": True})
+        report = profiler.report(
+            report_options={
+                "remove_disabled_flag": True,
+                "structured_options.multiprocess.is_enabled": False,
```

Comment:
Since this is just report, this won't have an effect and we can remove it from here.